Formula/task.rb

```diff
@@ -1,8 +1,8 @@
 class Task < Formula
   desc "Feature-rich console based todo list manager"
   homepage "https://taskwarrior.org/"
-  url "https://taskwarrior.org/download/task-2.5.1.tar.gz"
-  sha256 "d87bcee58106eb8a79b850e9abc153d98b79e00d50eade0d63917154984f2a15"
+  url "https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v2.5.3/task-2.5.3.tar.gz"
```

Comment:
Why the change in the source URL? If we're changing this, then the `livecheck` block should also be updated accordingly.

Reply:
Ok, but please update the livecheck block accordingly. I think you can just drop the `regex` line and change the `url` to `:stable`. CC @samford

Reply:
I will file a separate PR on the livecheck thing. :)

Reply:
#69175 will take care of updating the `livecheck` block to use the new `stable` source (GitHub). I figured I would save you the trouble, @chenrui333.

Reply:
Thanks @samford!